gen-apidocs/generators/api/config.go

```diff
@@ -557,3 +560,19 @@ func doScaleIdHack(version, name, match string) (string, string) {
 
 	return version, name
 }
+
+func (config *Config) createDescriptionsWithEntities () {
+
+	// The OpenAPI spec has escape sequences like \u003c. When the spec is unmarshaled,
+	// the escape sequences get converted to ordinary characters. For example,
+	// \u003c gets converted to a regular < character. But we can't use  regular <
+	// and > characters in our HTML document. This function replaces the regular
+	// < and > characters with HTML entities.
+	
+	for _, definition := range config.Definitions.GetAllDefinitions() {
+		d := definition.Description()
+		d = strings.Replace(d, "<", "&lt;", -1)
```

Comment:
Would html.EscapeString be more appropriate/robust here?

Reply:
Yes thanks. Changed to html.EscapeString.